skops/io/_general.py

```diff
@@ -427,6 +446,24 @@ def object_get_state(obj: Any, save_context: SaveContext) -> dict[str, Any]:
     return res
 
 
+class ConstructorFromReduceNode(Node):
+    def __init__(
+        self,
+        state: dict[str, Any],
+        load_context: LoadContext,
+        trusted: Optional[Sequence[str]] = None,
+    ) -> None:
+        super().__init__(state, load_context, trusted)
+        self.children = {
+            "content": get_tree(state["content"], load_context, trusted=trusted)
+        }
```

Comment:
Tbh I'm a bit rusty on this, do we need to set `self.trusted`?

Reply:
we pass trusted here if the user passes them, and sometimes we'd add to the trusted values if we have some default trusted types.